Cognite.Simulator.Utils/ConnectorBase.cs

```diff
@@ -32,6 +33,8 @@ public abstract class ConnectorBase
         private readonly ILogger<ConnectorBase> _logger;
         private readonly ConnectorConfig _config;
 
+        private string LastLicenseCheckTimestamp { get; set; } = "0";
```

Comment:
this should be long, convert it to string upon writing to the sequence

Reply:
If it should end as a string anyway, it seems easier for me to just make it as a string from the beginning

Reply:
> If it should end as a string anyway
This is just because the persistence structure we use only allows string at the moment.
Normally, the timestamp is in fact long, so we should keep it that way up until it is serialized.
This allows us to swap to a different persistence layer (which allows long values) in the future easily.

Overall it's better to not shoehorn the data based on serialisation/deserialisation logic if possible.
(otherwise everything would have been JSON for us) :)